app/src/constants/radioGroupThemes.ts

```diff
@@ -0,0 +1,32 @@
+interface Theme {
+  textClass: string;
+  containerClass: string;
+  circleClass: string;
+}
+
+interface Themes {
+  [key: string]: Theme;
+}
```

Comment:
Wdyt about changing `interfaces` to `Type` aliases? 🤔  
Like so: 
```typescript
type Theme = {
  textClass: string;
  containerClass: string;
  circleClass: string;
}

type Themes = {
  [key: string]: Theme;
}
```
I think if we ever need to extend the `Theme` we can upgrade from `Type` and make it an as `interface`. 
Does that sound good to you? 😄

Reply:
Yeah, I totally had interface in mind 😅